Tests/Algorithm/AlgorithmTradingTests.cs

```diff
@@ -91,7 +102,7 @@ public void SetHoldings_ZeroToShort(decimal leverage)
         public void SetHoldings_ZeroToShort_SmallConstantFeeStructure(decimal leverage)
         {
             Security msft;
-            var algo = GetAlgorithm(out msft, 1, 1);
+            var algo = GetAlgorithm(out msft, leverage, 1);
```

Comment:
Ah, you're completely correct again! The SetHoldings function is unlevered, so 1x is 1 times total portfolio value, so the leverage should only impact the required margin for the order, but the order should be the same. Perfect!